test/docker-e2e/e2e_test.go

```diff
@@ -22,6 +24,7 @@ import (
 const (
 	txsimImage = "ghcr.io/celestiaorg/txsim"
 	txSimTag   = "v4.0.7-mocha"
+	homeDir    = "/var/cosmos-chain/celestia"
```

Comment:
[nit][question] why not use the default celestia-app home dir: `~/.celestia-app` or `/var/.celestia-app`

Reply:
For me it is indifference. @chatton might have more context on answering this one.

Reply:
this is just the path that is configured internally as the home path on the docker containers. We could make this configurable if we wanted but right now it's not.

Reply:
if its required for the tests to work, can we make it a constant in tastora?